lib/lotus/model/migrator/adapter.rb

```diff
@@ -158,6 +159,12 @@ def options
           @connection.opts
         end
 
+        # @since 0.4.1
+        # @api private
+        def connection
```

Comment:
Sure, I'd like to keep it private, tough :)

On Sun, Aug 9, 2015 at 9:56 PM Trung Lê <notifications@github.com> wrote:

> In lib/lotus/model/migrator/adapter.rb
> <https://github.com/lotus/model/pull/220#discussion_r36596408>:
>
> > @@ -158,6 +159,12 @@ def options
> >            @connection.opts
> >          end
> >
> > +        # @since 0.4.1
> > +        # @api private
> > +        def connection
>
> can we do attr_reader instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lotus/model/pull/220/files#r36596408>.
>